stan/math/rev/fun/log_inv_logit.hpp

```diff
@@ -13,15 +13,32 @@ namespace math {
  * Return the natural logarithm of the inverse logit of the
  * specified argument.
  *
- * @param u argument
+ * @tparam T An arithmetic type
+ * @param u `var_value` with inner arithmetic type
  * @return log inverse logit of the argument
  */
-inline var log_inv_logit(const var& u) {
+template <typename T, require_arithmetic_t<T>* = nullptr>
+inline auto log_inv_logit(const var_value<T>& u) {
   return make_callback_var(log_inv_logit(u.val()), [u](auto& vi) mutable {
     u.adj() += vi.adj() * inv_logit(-u.val());
   });
 }
```

Comment:
This could be a single template that handles both cases:
```suggestion
template <typename T, require_t<disjunction<std::is_arithmetic<T>, is_eigen<T>>>* = nullptr>
inline auto log_inv_logit(const var_value<T>& u) {
  return make_callback_var(log_inv_logit(u.val()), [u](auto& vi) mutable {
    u.adj() += elt_multiply(vi.adj(), inv_logit(-u.val()));
  });
}
```